### ISSUE TITLE
Add support for `.tar.xz` format

### ISSUE THREAD

**Issue by hung-gcx** [grandcentrix/actions-zephyr-sdk#5]
Zephyr SDK version 0.16.0 (and newer) uses `.tar.xz` : https://github.com/zephyrproject-rtos/sdk-ng/releases
Would be great if this format is supported.

**Comment by M1cha** [comment]
Please test if it works with tag `test/xz`.

**Comment by hung-gcx** [comment]
I got this:
`2023-04-27T05:46:48.4377310Z Requested labels: ubuntu-latest
2023-04-27T05:46:48.4377615Z Job defined at: GCX-EMBEDDED/gcx-modbus-com-system-firmware/.github/workflows/essentials.yml@refs/heads/update-ci-dependencies
2023-04-27T05:46:48.4377757Z Waiting for a runner to pick up this job...
2023-04-27T05:46:48.7456009Z Job is waiting for a hosted runner to come online.
2023-04-27T05:46:51.8074537Z Job is about to start running on the hosted runner: GitHub Actions 2 (hosted)
2023-04-27T05:46:55.0737694Z Current runner version: '2.303.0'
2023-04-27T05:46:55.0769606Z ##[group]Operating System
2023-04-27T05:46:55.0770302Z Ubuntu
2023-04-27T05:46:55.0771111Z 22.04.2
2023-04-27T05:46:55.0771598Z LTS
2023-04-27T05:46:55.0772020Z ##[endgroup]
2023-04-27T05:46:55.0772437Z ##[group]Runner Image
2023-04-27T05:46:55.0772919Z Image: ubuntu-22.04
2023-04-27T05:46:55.0773501Z Version: 20230426.1
2023-04-27T05:46:55.0774158Z Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20230426.1/images/linux/Ubuntu2204-Readme.md
2023-04-27T05:46:55.0774969Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20230426.1
2023-04-27T05:46:55.0775542Z ##[endgroup]
2023-04-27T05:46:55.0776100Z ##[group]Runner Image Provisioner
2023-04-27T05:46:55.0776555Z 2.0.161.1
2023-04-27T05:46:55.0776946Z ##[endgroup]
2023-04-27T05:46:55.0778257Z ##[group]GITHUB_TOKEN Permissions
2023-04-27T05:46:55.0779098Z Actions: write
2023-04-27T05:46:55.0779508Z Checks: write
2023-04-27T05:46:55.0780239Z Contents: write
2023-04-27T05:46:55.0780840Z Deployments: write
2023-04-27T05:46:55.0781374Z Discussions: write
2023-04-27T05:46:55.0781787Z Issues: write
2023-04-27T05:46:55.0782188Z Metadata: read
2023-04-27T05:46:55.0782608Z Packages: write
2023-04-27T05:46:55.0783107Z Pages: write
2023-04-27T05:46:55.0783573Z PullRequests: write
2023-04-27T05:46:55.0784044Z RepositoryProjects: write
2023-04-27T05:46:55.0784555Z SecurityEvents: write
2023-04-27T05:46:55.0784984Z Statuses: write
2023-04-27T05:46:55.0785459Z ##[endgroup]
2023-04-27T05:46:55.0789828Z Secret source: Actions
2023-04-27T05:46:55.0790592Z Prepare workflow directory
2023-04-27T05:46:55.1854011Z Prepare all required actions
2023-04-27T05:46:55.2090852Z Getting action download info
2023-04-27T05:46:55.5465899Z Download action repository 'grandcentrix/actions-ssh2https@v1' (SHA:6fec85fda82c9971ed8976a646c40f35b76b4f13)
2023-04-27T05:46:56.0700261Z Download action repository 'grandcentrix/actions-zephyr-sdk@test/xz' (SHA:035feeb1e2c5c68e29dec6097025e0dc5168ec7b)
2023-04-27T05:46:56.5786100Z Download action repository 'actions/cache@v3' (SHA:88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8)
2023-04-27T05:46:57.2027372Z Download action repository 'actions/upload-artifact@v3' (SHA:0b7f8abb1508181956e8e162db84b466c27e18ce)
2023-04-27T05:46:57.6529005Z Download action repository 'emibcn/badge-action@v2.0.2' (SHA:4209421db54c8764d8932070ffd0f81715a629bf)
2023-04-27T05:46:58.2544922Z ##[error]System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node18' is not supported, use 'docker', 'node12' or 'node16' instead.')
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
2023-04-27T05:46:58.2581098Z ##[error]Fail to load grandcentrix/actions-zephyr-sdk/test/xz/action.yml`

**Comment by hung-gcx** [comment]
I got this:
```
2023-04-27T05:46:48.4377310Z Requested labels: ubuntu-latest
2023-04-27T05:46:48.4377615Z Job defined at: GCX-EMBEDDED/gcx-modbus-com-system-firmware/.github/workflows/essentials.yml@refs/heads/update-ci-dependencies
2023-04-27T05:46:48.4377757Z Waiting for a runner to pick up this job...
2023-04-27T05:46:48.7456009Z Job is waiting for a hosted runner to come online.
2023-04-27T05:46:51.8074537Z Job is about to start running on the hosted runner: GitHub Actions 2 (hosted)
2023-04-27T05:46:55.0737694Z Current runner version: '2.303.0'
2023-04-27T05:46:55.0769606Z ##[group]Operating System
2023-04-27T05:46:55.0770302Z Ubuntu
2023-04-27T05:46:55.0771111Z 22.04.2
2023-04-27T05:46:55.0771598Z LTS
2023-04-27T05:46:55.0772020Z ##[endgroup]
2023-04-27T05:46:55.0772437Z ##[group]Runner Image
2023-04-27T05:46:55.0772919Z Image: ubuntu-22.04
2023-04-27T05:46:55.0773501Z Version: 20230426.1
2023-04-27T05:46:55.0774158Z Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20230426.1/images/linux/Ubuntu2204-Readme.md
2023-04-27T05:46:55.0774969Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20230426.1
2023-04-27T05:46:55.0775542Z ##[endgroup]
2023-04-27T05:46:55.0776100Z ##[group]Runner Image Provisioner
2023-04-27T05:46:55.0776555Z 2.0.161.1
2023-04-27T05:46:55.0776946Z ##[endgroup]
2023-04-27T05:46:55.0778257Z ##[group]GITHUB_TOKEN Permissions
2023-04-27T05:46:55.0779098Z Actions: write
2023-04-27T05:46:55.0779508Z Checks: write
2023-04-27T05:46:55.0780239Z Contents: write
2023-04-27T05:46:55.0780840Z Deployments: write
2023-04-27T05:46:55.0781374Z Discussions: write
2023-04-27T05:46:55.0781787Z Issues: write
2023-04-27T05:46:55.0782188Z Metadata: read
2023-04-27T05:46:55.0782608Z Packages: write
2023-04-27T05:46:55.0783107Z Pages: write
2023-04-27T05:46:55.0783573Z PullRequests: write
2023-04-27T05:46:55.0784044Z RepositoryProjects: write
2023-04-27T05:46:55.0784555Z SecurityEvents: write
2023-04-27T05:46:55.0784984Z Statuses: write
2023-04-27T05:46:55.0785459Z ##[endgroup]
2023-04-27T05:46:55.0789828Z Secret source: Actions
2023-04-27T05:46:55.0790592Z Prepare workflow directory
2023-04-27T05:46:55.1854011Z Prepare all required actions
2023-04-27T05:46:55.2090852Z Getting action download info
2023-04-27T05:46:55.5465899Z Download action repository 'grandcentrix/actions-ssh2https@v1' (SHA:6fec85fda82c9971ed8976a646c40f35b76b4f13)
2023-04-27T05:46:56.0700261Z Download action repository 'grandcentrix/actions-zephyr-sdk@test/xz' (SHA:035feeb1e2c5c68e29dec6097025e0dc5168ec7b)
2023-04-27T05:46:56.5786100Z Download action repository 'actions/cache@v3' (SHA:88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8)
2023-04-27T05:46:57.2027372Z Download action repository 'actions/upload-artifact@v3' (SHA:0b7f8abb1508181956e8e162db84b466c27e18ce)
2023-04-27T05:46:57.6529005Z Download action repository 'emibcn/badge-action@v2.0.2' (SHA:4209421db54c8764d8932070ffd0f81715a629bf)
2023-04-27T05:46:58.2544922Z ##[error]System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node18' is not supported, use 'docker', 'node12' or 'node16' instead.')
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
2023-04-27T05:46:58.2581098Z ##[error]Fail to load grandcentrix/actions-zephyr-sdk/test/xz/action.yml
```

**Comment by M1cha** [comment]
I downgraded to node16, please try again

**Comment by hung-gcx** [comment]
Works! (:

**Comment by M1cha** [comment]
The fix was released to `v1`, `v1.3` and `latest`.
